arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -2894,6 +2894,12 @@ DECLARE_GAUGE(rocksdb_block_cache_capacity, uint64_t,
 DECLARE_GAUGE(rocksdb_block_cache_pinned_usage, uint64_t,
               "rocksdb_block_cache_pinned_usage");
 DECLARE_GAUGE(rocksdb_block_cache_usage, uint64_t, "rocksdb_block_cache_usage");
+#ifdef ARANGODB_ROCKSDB8
+// DECLARE_GAUGE(rocksdb_block_cache_entries, uint64_t,
+//                    "rocksdb_block_cache_entries");
+// DECLARE_GAUGE(rocksdb_block_cache_charge_per_entry, uint64_t,
+//                    "rocksdb_block_cache_charge_per_entry");
+#endif
```

Comment:
Should we remove the comments since this is already ifdefed?

Reply:
The comments need to be present, because without them the `check-guidelines` check in Jenkins will fail.
This check verifies that for all declared metrics there is a corresponding file in `Documentation`. The comments disable the check for these metrics.